psychopy/visual/noise.py

```diff
@@ -35,7 +35,6 @@
 import numpy
 from numpy import exp, sin, cos
 from numpy.fft import fft2, ifft2, fftshift, ifftshift
-from scipy.misc import imresize, imrotate
```

Comment:
@schofiaj the new versions of scipy have dropped support for these (but they were previously using PIL before apparently) so all the changes below are about handling that change for future

Reply:
Although the code looks longer I think the compute time will be the same (becuase I think this is probably exactly what scipy.imresize was actually doing!)

Reply:
Thanks Jon, I'd noticed that these image functions were depreciated in scipy and had it on my list to recode to avoid them. Hopefully your changes will deal with that approriately.